.theme/nuxt.config.ts

```diff
@@ -11,6 +11,32 @@ export default defineNuxtConfig({
       },
     }
   },
+  app: {
+    head: {
+      meta: [
+        {
+          hid: "og:description",
+          property: "og:description",
+          content: "Welcome to >Mouseless_",
```

Comment:
bundan bahsetmemistim, typo yapmissin mousless yazmistin, mouseless demen yeterli;
https://github.com/mouseless/brand#usage-in-plain-text

text logo olacaksa da boyle degil soyle;
https://github.com/mouseless/brand#logo-usage-in-plain-text

Reply:
logo.png yi logo-[form]-[color]-[height].png bu formatta rename edeyim mi o zaman ?

Reply:
hayır o o repoyla ilgili